examples/text-generation/utils.py

```diff
@@ -181,6 +181,34 @@ def get_torch_compiled_model(model):
     return model
 
 
+def setup_quantization(model, args):
+    if os.getenv("USE_INC", "1") != "0":
+        from neural_compressor.torch.quantization import FP8Config, convert, prepare
```

Comment:
When I run I see error:
  File "/root/PR/holyfalafel_inc/examples/text-generation/utils.py", line 186, in setup_quantization
    from neural_compressor.torch.quantization import FP8Config, convert, prepare
ModuleNotFoundError: No module named 'neural_compressor'

I ran on v1.17.0 container and install optimum-habana, Deepspeed, requirements for text-generation. What am I missing?

Reply:
which build of 1.17.0 is your container based on? I took container 1.17.0-484 now and it worked.

Reply:
Okay it works with build 484, I was on an older build earlier.
I am concerned about that folks on earlier builds of v1.17.0 (and we have customers on v1.16.0 still) will see an issue if they don't use the recent containers. You could do this:

Please protect the imports of nerual_compressor with try and except ImportError (you might need to negative test to check this works) , report error and inform user to set USE_INC=0 to enable HQT as INC is not supported in the container or use a more recent container.

Reply:
Done